static/js/util/util_test.js

```diff
@@ -324,12 +325,12 @@ describe('utility functions', () => {
       profile = {};
     });
 
-    it('should return fields for dialog for an empty profile', () => {
-      let expectation = [false, {
-        url: "/profile/personal",
-        title: "Personal Info",
-        text: "Please complete your personal information.",
-      }];
+    it('should return fields for an empty profile', () => {
+      let errors = {};
+      _.forIn(PersonalTab.defaultProps.validationMessages, (value, key) => {
+        _.set(errors, key, `${value} is required`);
+      });
```

Comment:
this works, but I think this might be cleaner:

```js
let errors = Object.assign({}, ...Object.entries(PersonalTab.defaultProps.validationMessages).map(
  ([k,v]) => ({[k]: `${v} is required`})
));
```

we'd need to add the `Object.entries` polyfill though, by adding the following to `static/js/global_init.js`:

```js
import entries from 'object.entries';
if (!Object.entries) {
  entries.shim();
}
```

your call.